### DISCUSSION TITLE
failed to extract shortcode: template for shortcode "gallery" not found

### DISCUSSION THREAD

**Discussion by klische** [thuliteio/doks#273] [Q&A]
## Description

After cloning this theme into the /themes directory, I get the error: failed to extract shortcode: template for shortcode "gallery" not found

### Steps to reproduce

1. Clone the hugo sample code 
2. Check that the sample hugo site is working
3. Clone the doks repo
4. Update the toml file to use the doks theme
5. Restart the hugo server, it fails with the error message above 
### Expected result
It should work
What should happen?

### Actual result
Results in error message above
What happened.

### Environment

Paste the information here as shown by `npm run check`

**Comment by h-enk** [comment]
Doks uses npm to centralize dependency management, making it easy to update resources, build tooling, plugins, and build scripts. Because of that, Doks needs to live in the root and does not work from the `./themes` directory.
